catalyst/contrib/scripts/project_embeddings.py

```diff
@@ -6,7 +6,13 @@
 import numpy as np
 import pandas as pd
 import torch
-from torch.utils.tensorboard import SummaryWriter
+
+# Native tensorboard support from 1.2.0 version of PyTorch
```

Comment:
can we move it to some utils module?

and in all the other places, `from catalyst.utils.tensorboard import SummaryWriter`

Reply:
Good idea, I'll move this if construction to catalyst.utils.tensorboard.